### ISSUE TITLE
Add support for WebVTT subtitles embedded in webm files

### ISSUE THREAD

**Issue by ale5000-git** [pinterf/xy-VSFilter#19]
Sample file: https://gofile.io/d/2jbaFN
This fork of LAVFilters support it:
https://github.com/clsid2/LAVFilters

It is included in K-Lite Codec Pack 16.0.9.

Ref: https://github.com/pinterf/xy-VSFilter/issues/18

Could you please add support for WebVTT subtitles embedded in webm files?

**Comment by pinterf** [comment]
Well, O.K., I made it to recognize the WebVTT format but no formatting is supported. (and as I see neither does it MPC-HC with the above sample)
It would be nice that as a minimum formatter tags must to be omitted. Leaving the bare text.
Info for myself: https://w3c.github.io/webvtt/#webvtt-cue-class-span
Writing a parser for that is not a two minutes task - at least not for me :)

**Comment by clsid2** [comment]
MPC-HC converts a few tags to SSA format, and strips a few unsupported ones.

For example see last few commits from today:
https://github.com/clsid2/mpc-hc/commits/develop

**Comment by pinterf** [comment]
Great, thank you clsid2, it's a hot topic, I'll then wait a bit.

**Comment by ale5000-git** [comment]
@pinterf: The latest MPC-HC plus [this pull request](https://github.com/clsid2/mpc-hc/pull/1026) already has a considerable support.

**Comment by pinterf** [comment]
Yep, the earlier changes were already integrated, still waiting a bit to settle down.

**Comment by ale5000-git** [comment]
Can this be considered done or there is something missing?

**Comment by pinterf** [comment]
I think it is considered done, except that I haven't checked since March whether it was developed further in MPC-HC.
